### ISSUE TITLE
if set keyup: true, keydown can not trigger

### ISSUE THREAD

**Issue by yyjjqq94** [jaywcjlove/hotkeys#67]
code like this:

```js
hotkeys('a', { keyup: true }, (evn, handler) => {
  if(evn.type === 'keydown') {
    // this part can not trigger 
    console.log('keydown:', evn.type, handler, handler.key);
  }
  if(evn.type === 'keyup') {
    console.log('keyup:', evn.type, handler, handler.key);
  }
});
```

so i need set keyup: true to handdle keyup event,and write another hotkey(...) to handdle keydown event

**Comment by jaywcjlove** [comment]
@yyjjqq94 thx! already fixed, Upgrade `hotkeys-js@3.6.3`

**Comment by jaywcjlove** [comment]
Example: https://codepen.io/jaywcjlove/pen/pBYygV
